google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/converter/AppEngineFacetMigrationConversionProcessor.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.appengine.converter;
```

Comment:
'converter' is usually referring to pretty generic bean conversion via types like Guava Converter.  Maybe 'migration[s]' might be a more descriptive package?